app/controllers/admin/partnerships_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module Admin
+  class PartnershipsController < Admin::BaseController
+    skip_after_action :verify_policy_scoped
+    before_action :load_partnership, only: %i[show reject]
+
+    def show
+      authorize @partnership
+      @reasons_for_rejection = [
+        OpenStruct.new(id: :partnered_with_another_provider, name: "I have already partnered with a different training provider"),
```

Comment:
We could make it into a solid class as opposed to using openstruct

Reply:
I don't think that would be a good idea. This is quite simple, it has no behaviour just some data and some convenient accessors. I can't see any justification for making this a full blown class. And there's always the testing overhead that comes with new units like classes.